Bio/PDB/FragmentMapper.py

```diff
@@ -18,7 +18,7 @@
 
 The definition files of the fragments can be obtained from:
 
-U{http://csb.stanford.edu/~rachel/fragments/}
+U{http://github.com/csblab/fragments/}
```

Comment:
That weird ``U{...}`` markup is not valid RST, and we should fix that while you're working on this. The simplest fix is to make it a plain URL.

I find http://rst.ninjs.org/ useful for RST checking.